### PR TITLE
Issue #375

### PR DESCRIPTION
**Pull request by kirjaamo**
mysqld_bootstrap --wsrep-new-cluster to mysqld_bootstrap

### CODE REVIEW

**Comment by kirjaamo** [comment]
I hope I got it right...

File: galeracluster/source/documentation/jails.rst [hunk 1]
```diff
@@ -307,7 +307,7 @@ To start the initial cluster node, run the following commands:
 .. code-block:: console
 
    # ezjail-admin console galera-node
-   # service mysql start --wsrep-new-cluster
+   # service mysql start mysqld_bootstrap
```
Comment (byte) [inline galeracluster/source/documentation/jails.rst:310]: I don't think this is right for FreeBSD. Can you check, failing which, don't change this yet, by filing another issue

File: galeracluster/source/documentation/mysql-wsrep-options.rst [hunk 2]
```diff
@@ -361,9 +361,9 @@ Changing this variable while Galera is running will cause the node to close the
...
 
-.. warning:: Never use an empty ``gcomm://`` string with the ``wsrep_cluster_address`` option in the configuration file. If a node restarts, it will cause the node not to rejoin the cluster. Instead, it will initialize a new one-node cluster and cause a :term:`Split Brain`. To bootstrap a cluster, you should only pass the ``--wsrep-new-cluster`` string at the command-line--instead of using ``--wsrep-cluster-address="gcomm://"``. For more information, see :doc:`Starting the Cluster <../training/tutorials/starting-cluster>`.
+.. warning:: Never use an empty ``gcomm://`` string with the ``wsrep_cluster_address`` option in the configuration file. If a node restarts, it will cause the node not to rejoin the cluster. Instead, it will initialize a new one-node cluster and cause a :term:`Split Brain`. To bootstrap a cluster, you should only pass the ``mysqld_bootstrap`` string at the command-line--instead of using ``--wsrep-cluster-address="gcomm://"``. For more information, see :doc:`Starting the Cluster <../training/tutorials/starting-cluster>`.
```
Comment (byte) [inline galeracluster/source/documentation/mysql-wsrep-options.rst:366]: Nope

File: galeracluster/source/documentation/mysql-wsrep-options.rst [hunk 3]
```diff
@@ -361,9 +361,9 @@ Changing this variable while Galera is running will cause the node to close the
 
     gcomm://node1:port1,node2:port2,...[?option1=value1&...]
 
-Using the string ``gcomm://`` without any address will cause the node to startup alone, thus initializing a new cluster--that the other nodes can join to.  Using ``--wsrep-new-cluster`` is the newer, preferred way.
+Using the string ``gcomm://`` without any address will cause the node to startup alone, thus initializing a new cluster--that the other nodes can join to.  Using ``mysqld_bootstrap`` is the newer, preferred way.
```
Comment (byte) [inline galeracluster/source/documentation/mysql-wsrep-options.rst:364]: I think this also stays

File: galeracluster/source/documentation/quorum-reset.rst [hunk 4]
```diff
@@ -176,17 +176,17 @@ For servers that use ``systemd``, execute instead this from the command-line:
...
 
-You're now ready to start the cluster again.  Start the most advanced node with the ``--wsrep-new-cluster`` option |---| not the other nodes.  For servers that use ``init``, run the following command:
+You're now ready to start the cluster again.  Start the most advanced node with the ``mysqld_bootstrap`` command |---| not the other nodes.  For servers that use ``init``, run the following command:
 
 .. code-block:: console
 
-   service mysql start --wsrep-new-cluster
+   service mysql start mysqld_bootstrap
 
 For servers that use ``systemd`` and Galera Cluster 5.5 or 5.6, enter instead the following from the command-line:
```
Comment (byte) [inline galeracluster/source/documentation/quorum-reset.rst:185]: Considering we haven't supported 5.5 and 5.6 for quite sometime, I suggest we start focusing on 5.7 and 8.0

File: galeracluster/source/documentation/quorum-reset.rst [hunk 5]
```diff
@@ -176,17 +176,17 @@ For servers that use ``systemd``, execute instead this from the command-line:
...
 
 .. code-block:: console
 
-   systemctl start mysqld --wsrep-new-cluster
+   systemctl start mysqld mysqld_bootstrap
```
Comment (byte) [inline galeracluster/source/documentation/quorum-reset.rst:189]: systemctl start mysqld --wsrep-new-cluster OR mysqld_bootstrap 

not both

File: galeracluster/source/documentation/quorum-reset.rst [hunk 6]
```diff
@@ -214,7 +214,7 @@ For servers that use ``systemd``, instead run this command:
 
    systemctl start mysqld
 
-Written into all of these scripts is the ``--wsrep-new-cluster`` option, but it's done with a certain finesse.  Whichever method or script you use, when the first node starts with the ``--wsrep-new-cluster`` option, it initializes a new cluster using the data from the most advanced state available from the previous cluster.  As the other nodes start, they connect to this node and request state snapshot transfers, to bring their own databases up-to-date.  In a short amount of time, they all should become synchronized and running smoothly.
+Written into all of these scripts is the ``mysqld_bootstrap`` command, but it's done with a certain finesse.  Whichever method or script you use, when the first node starts with the ``mysqld_bootstrap`` command, it initializes a new cluster using the data from the most advanced state available from the previous cluster.  As the other nodes start, they connect to this node and request state snapshot transfers, to bring their own databases up-to-date.  In a short amount of time, they all should become synchronized and running smoothly.
```
Comment (byte) [inline galeracluster/source/documentation/quorum-reset.rst:217]: Nope

File: galeracluster/source/training/tutorials/starting-cluster.rst [hunk 7]
```diff
@@ -92,31 +92,31 @@ Once you have at least three hosts ready, you can initialize the cluster.
...
 
-   mysqld_bootstrap --wsrep-new-cluster
+   mysqld_bootstrap
 
 On MariaDB:
 
 .. code-block:: console
 
-   mysqld --wsrep-new-cluster
+   mysqld mysqld_bootstrap
```
Comment (byte) [inline galeracluster/source/training/tutorials/starting-cluster.rst:111]: No. 

Technically you can use galera_new_cluster

ref: https://mariadb.com/kb/en/getting-started-with-mariadb-galera-cluster/

File: galeracluster/source/documentation/jails.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/jails.rst:310]: Returned to the original. I cannot check, I don't the environment.

File: galeracluster/source/documentation/mysql-wsrep-options.rst (hunk 2 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/mysql-wsrep-options.rst:366]: Returned to original

File: galeracluster/source/documentation/mysql-wsrep-options.rst (hunk 3 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/mysql-wsrep-options.rst:364]: Returned to original

File: galeracluster/source/documentation/quorum-reset.rst (hunk 4 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/quorum-reset.rst:185]: Changed

File: galeracluster/source/documentation/quorum-reset.rst (hunk 5 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/quorum-reset.rst:189]: Fixed

File: galeracluster/source/documentation/quorum-reset.rst (hunk 6 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/quorum-reset.rst:217]: REturned to original

File: galeracluster/source/training/tutorials/starting-cluster.rst (hunk 7 above)
Comment (kirjaamo) [reply galeracluster/source/training/tutorials/starting-cluster.rst:111]: Back to original

**Pull request merged**